tmt/steps/execute/__init__.py

```diff
@@ -340,6 +343,51 @@ def check_beakerlib(self, test):
             data['result'] = result.lower()
         return tmt.Result(data, test.name, test.result)
 
+    def check_result_file(self, test):
+        """
+        Check result file created by tmt-report-result
+
+        Check to see if the rstrnt-report-result script was used
+        and has generated a report. If yes then check the result
+        and update the test result, otherwise use the original
+        result returned from the execute function.
```

Comment:
This is misleading  because in reality this method raises  tmt.utils.FileError in such case

Reply:
Good point, reworded.